### PR TITLE
[orchagent]: Publish identified events via structured-events channel

### PR DESCRIPTION
**Pull request by renukamanavalan**
**What I did**
Ref: https://github.com/sonic-net/sonic/pull/954
Three events identified are now published via events channel.

**Why I did it**
To publish events per HLD.

**How I verified it**
When the event occurs, 
1) A syslog line starting "EVENT_PUBLISHED:...." can be seen.
2) You may run a gNMI query anbd/or events_tool to capture the events.

**Details if related**

### CODE REVIEW

**Review by zbud-msft — APPROVED** [review]

**Review by zbud-msft — DISMISSED** [review]

File: orchagent/orchdaemon.cpp [hunk 1]
```diff
@@ -97,7 +100,9 @@ bool OrchDaemon::init()
 
     string platform = getenv("platform") ? getenv("platform") : "";
 
-    gCrmOrch = new CrmOrch(m_configDb, CFG_CRM_TABLE_NAME);
+    m_events_handle = events_init_publisher("sonic-events-swss");
```
[the remark this answers is not in the document]
Comment (zbud-msft) [reply orchagent/orchdaemon.cpp:103]: Would it be better to declare this as extern in orch.h or in the .cpp files that uses it?

[the remark this answers is not in the document]
Comment (renukamanavalan) [reply orchagent/orchdaemon.cpp:103]: > Please have this as a global variable so that we don't have to pass for each orch and change the signature in all places.

This is the right approach. Thanks!

[the remark this answers is not in the document]
Comment (prsunny) [reply orchagent/orchdaemon.cpp:103]: extern in the .cpp file that uses it

[the remark this answers is not in the document]
Comment (zbud-msft) [reply orchagent/orchdaemon.cpp]: Done

File: orchagent/p4orch/tests/fake_crmorch.cpp [hunk 2]
```diff
@@ -3,7 +3,7 @@
 
 #include "crmorch.h"
 
-CrmOrch::CrmOrch(swss::DBConnector *db, std::string tableName) : Orch(db, std::vector<std::string>{})
+CrmOrch::CrmOrch(swss::DBConnector *db, std::string tableName, event_handle_t) : Orch(db, std::vector<std::string>{})
```
Comment (prsunny) [inline orchagent/p4orch/tests/fake_crmorch.cpp:6]: You may want to revert this

File: orchagent/portsorch.cpp [hunk 3]
```diff
@@ -352,7 +353,8 @@ static bool isValidPortTypeForLagMember(const Port& port)
  *    bridge. By design, SONiC switch starts with all bridge ports removed from
  *    default VLAN and all ports removed from .1Q bridge.
  */
-PortsOrch::PortsOrch(DBConnector *db, DBConnector *stateDb, vector<table_name_with_pri_t> &tableNames, DBConnector *chassisAppDb) :
+PortsOrch::PortsOrch(DBConnector *db, DBConnector *stateDb, vector<table_name_with_pri_t> &tableNames,
```
Comment (prsunny) [inline orchagent/portsorch.cpp:356]: Can you please revert this change if its not modified?

File: orchagent/p4orch/tests/fake_crmorch.cpp (hunk 2 above)
Comment (zbud-msft) [reply orchagent/p4orch/tests/fake_crmorch.cpp]: Done

File: orchagent/portsorch.cpp (hunk 3 above)
Comment (zbud-msft) [reply orchagent/portsorch.cpp]: Done

**Review by prsunny — APPROVED** [review]
